playbook.yml

```diff
@@ -15,5 +15,4 @@
         name: libselinux-python3
         state: present
   roles:
-    - ansible-keylime-tpm20
-    - ansible-keylime-user
+    - ansible-keylime-tpm20
```

Comment:
Check your editor settings. GitHub is indicating that a newline is missing at the end of file.